PocketKit/Sources/Sync/PocketSource.swift

```diff
@@ -464,10 +475,16 @@ extension PocketSource {
                 )
                 enqueue(operation: operation, persistentTask: persistentTask)
             case .addTags(let remoteID, let tags):
+                let mutation: AnyMutation
+                if tags.isEmpty {
```

Comment:
Not strictly necessary, but could extract this bit of conditional logic into a private method so that we don't have to repeat this code here _and_ in `addTags()`

Reply:
will do